index.html

```diff
@@ -9437,9 +9437,9 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
       <code>RTCDTMFSender</code> interface, whereas ORTC originally supported
       the <code>setTrack</code> method and the <code>RTCDtmfSender</code>
       interface. In order to provide backward compatibility, this specification has
-      added <code>replaceTrack</code> as an alias for <code>setTrack</code>,
-      and the <code>RTCDTMFSender</code> interface as an alias for the
-      <code>RTCDtmfSender</code> interface.</p>
+      added <code>replaceTrack</code> as an alias for <code>setTrack</code>.
+      and the <code>RTCDTMFSender</code> interface, which should be treated
+      as an alias for <code>RTCDtmfSender</code> interface (now deprecated).</p>
```

Comment:
Repeating https://github.com/w3c/ortc/pull/661#discussion_r111724617: The wording in https://drafts.fxtf.org/geometry/#DOMMatrix is good: "For historical reasons, Window objects must also have writable, configurable, non-enumerable properties named SVGMatrix and WebKitCSSMatrix whose value is the DOMMatrix interface object."